### PR TITLE
#3416 Intake skeleton

### PR DESCRIPTION
**Pull request by NickHeiner**
I took a selfie when I was writing the Rails part of this PR:

image

### CODE REVIEW

File: client/app/intake/pages/second.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class First extends React.PureComponent {
```
Comment (shanear) [inline client/app/intake/pages/second.jsx]: `Second`?

File: client/app/reader/DecisionReviewer.jsx [hunk 2]
```diff
@@ -145,7 +145,7 @@ export class DecisionReviewer extends React.PureComponent {
                 render={this.routedPdfListView}/>
               <PageRoute
                 exact
-                title ="Document Viewer | Caseflow Reader"
+                title="Document Viewer | Caseflow Reader"
```
Comment (shanear) [inline client/app/reader/DecisionReviewer.jsx]: little side refactor here lol

File: client/app/components/RadioField.jsx [hunk 3]
```diff
@@ -28,6 +28,7 @@ export default class RadioField extends React.Component {
       onChange,
       required,
       errorMessage,
+      strongLabel,
```
Comment (shanear) [inline client/app/components/RadioField.jsx]: shouldn't all our labels be either strong or not?

File: client/app/intake/pages/review.jsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      { displayText: 'Supplemental Claim' },
+      { displayElem: <span>Higher Level Review <strong>with</strong> DRO hearing request</span> },
+      { displayElem: <span>Higher Level Review with<strong>out</strong> DRO hearing request</span> },
+      { displayText: 'Withdraw all pending appeals' }
+    ];
+
+    return <div>
+      <h1>Review Joe Snuffy's opt-in request</h1>
+      <p>Check the Veteran's RAMP Opt-In Election form in the Centralized Portal.</p>
+      <RadioField
```
Comment (shanear) [inline client/app/intake/pages/review.jsx]: i ❤️ style guide

File: client/app/intake/pages/review.jsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import RadioField from '../../components/RadioField';
+import DateSelector from '../../components/DateSelector';
+
+export default class Review extends React.PureComponent {
+  onElectionChange = () => {
```
Comment (shanear) [inline client/app/intake/pages/review.jsx]: were planning on going with redux for this right?

Comment (NickHeiner) [reply client/app/intake/pages/review.jsx]: Yes. `RadioField` complains if you don't pass it an `onChange` handler, so I created this empty function.

File: client/app/intake/pages/review.jsx (hunk 4 above)
Comment (NickHeiner) [reply client/app/intake/pages/review.jsx]: 💪

File: client/app/components/RadioField.jsx (hunk 3 above)
Comment (NickHeiner) [reply client/app/components/RadioField.jsx]: By default, `RadioField` does not create `strong` labels. The mockups @abbyraskinUSDS provided had `strong` labels. I assumed this was an intentional difference. :smile:

**Pull request merged**